src/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './PageHeader.module.scss'
+import logo from '../../img/logo.png'
```

Comment:
do not use common folder for assets.
You should follow the next structure
PageHeader
-- assets
-- -- logo.png
-- PageHeader.jsx
-- PageHeader.module.scss

`import logo from './assets/logo.png'`